temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -221,9 +208,12 @@ public Builder setTarget(String target) {
       return this;
     }
 
-    /** Global test timeout. Default is 10 seconds. */
+    /**
+     * @deprecated Temporal test rule shouldn't be responsible for enforcing test timeouts. Use
+     *     toolchain of your test framework to enforce timeouts.
```

Comment:
Should mention not only is this deprecated, it now does nothing.

Reply:
What about deprecating it for now, but setting timeout only if the value is set?

Reply:
I rolled back the removal of the actual implementation. Let's clean it out in 1.5